### ISSUE TITLE
[Shared] selectAction title should be passed to UI automation for accessibility

### ISSUE THREAD

**Issue by andrewleader** [Microsoft/AdaptiveCards#2425]
**Automatically generated task. Any updates to the title and body will be overwritten. Do NOT modify the labels.**

Shared implementation of #1298

**Comment by andrewleader** [comment]
Closing since irrelevant to shared model (UI feature only)
